backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -269,6 +204,121 @@ public void addEmail(Email email) {
                 });
     }
 
+    private Uni<String> renderBody(Notification item, EmailSubscriptionType emailSubscriptionType, EmailTemplate emailTemplate) {
+        return emailTemplate.getBody(item.getAction().getEventType(), emailSubscriptionType)
+                .data("action", item.getAction())
+                .createMulti()
+                .collect().with(Collectors.joining())
+                .onFailure()
+                .recoverWithItem(templateEx -> {
+                    log.log(Level.WARNING, templateEx, () -> String.format(
+                            "Unable to render template body for application: [%s], eventType: [%s], subscriptionType: [%s].",
+                            item.getAction().getApplication(),
+                            item.getAction().getEventType(),
+                            emailSubscriptionType
+                    ));
+                    return null;
+                });
+    }
+
+    private Uni<String> renderTitle(Notification item, EmailSubscriptionType emailSubscriptionType, EmailTemplate emailTemplate) {
+        return emailTemplate.getTitle(item.getAction().getEventType(), emailSubscriptionType)
+                .data("action", item.getAction())
+                .createMulti()
+                .collect().with(Collectors.joining())
+                .onFailure()
+                .recoverWithItem(templateEx -> {
+                    log.log(Level.WARNING, templateEx, () -> String.format(
+                            "Unable to render template title for application: [%s], eventType: [%s], subscriptionType: [%s].",
+                            item.getAction().getApplication(),
+                            item.getAction().getEventType(),
+                            emailSubscriptionType
+                    ));
+                    return null;
+                });
+    }
+
+    private Uni<Tuple3<Email, String, String>> combine(Email email, Uni<String> title, Uni<String> body) {
+        return Uni.combine().all()
+                .unis(
+                        Uni.createFrom().item(email),
+                        title,
+                        body
+                ).asTuple()
+                .onItem().transform(objects -> {
+                    if (objects == null || objects.getItem1() == null || objects.getItem2() == null || objects.getItem3() == null) {
+                        return null;
+                    }
+
+                    return objects;
+                });
+    }
+
+    @Incoming(AGGREGATION_CHANNEL)
+    public Uni<Void> consumeEmailAggregations(String aggregationsJson) {
+        List<EmailAggregation> aggregations;
+        try {
+            aggregations = objectMapper.readValue(aggregationsJson, new TypeReference<>() {
+            });
+        } catch (JsonProcessingException e) {
+            log.info(e.getMessage());
```

Comment:
We need the Sentry alert with a full stack trace if that happens:
```suggestion
            log.error("Kafka aggregation payload parsing failed", e);
```